typescript/infra/src/agents/aws/user.ts

```diff
@@ -114,6 +116,33 @@ export class AgentAwsUser<Chain extends ChainName> {
     return key;
   }
 
+  private async getUsers(): Promise<User[]> {
```

Comment:
A single `ListUsersCommand` returns at most 100 users. We create a new user for each agent (so they can access their specific key, or in the case of a relayer, they very annoyingly need to be an AWS user to use the Node.js AWS SDK to read from S3 buckets even if they're public). We recently got over the 100 mark for total # of accounts, so to list all the users we have to paginate

As to why we are listing users to begin with instead of trying to get the specific user using the SDK, I don't 100% remember but it was probably to do with planning around permissions. Most of us will have permission to list metadata about users, but not all of us will have permission to get specific user's detailed information